wp-content/themes/pub/wporg-learn-2024/src/style/_sensei.scss

```diff
@@ -73,6 +73,97 @@ body.sensei {
 			font-size: var(--wp--preset--font-size--heading-2);
 		}
 	}
+
+	section.wp-block-sensei-lms-course-outline {
+		> section.wp-block-sensei-lms-course-outline-module-bordered {
+			border: 1px solid var(--wp--preset--color--light-grey-1) !important;
+			border-radius: 2px;
+
+			header {
+				background-color: var(--wp--preset--color--light-grey-2) !important;
+				color: var(--wp--preset--color--charcoal-1) !important;
+				padding: var(--wp--preset--spacing--20);
+
+				h2 {
+					font-size: var(--wp--preset--font-size--heading-6);
+					font-weight: var(--wp--custom--button--typography--font-weight);
+				}
+
+				div.wp-block-sensei-lms-course-outline-module__progress-indicator {
+					border-radius: 2px;
+
+					&.completed {
+						color: var(--wp--custom--color--green-50) !important;
+					}
+
+					.wp-block-sensei-lms-course-outline-module__progress-indicator__text {
+						font-family: "SF Pro Text", sans-serif;
+						font-size: 11px;
+						font-weight: var(--wp--custom--button--typography--font-weight);
+						line-height: var(--wp--custom--body--huge--typography--line-height);
+						margin: 4px 8px;
+					}
+				}
+			}
+
+			.wp-block-sensei-lms-course-outline-module__lessons-title {
+				display: none;
+			}
+
+			.wp-block-sensei-lms-collapsible {
+				padding-top: 5px;
+				padding-bottom: 5px;
+			}
+		}
+
+		// every lesson item
+		.wp-block-sensei-lms-course-outline-lesson {
+			> span {
+				line-height: var(--wp--custom--body--short-text--typography--line-height);
+				padding: 15px var(--wp--preset--spacing--10);
+			}
+
+			> svg {
+				margin-left: var(--wp--preset--spacing--20);
+			}
+
+			> .wp-block-sensei-lms-course-outline-lesson__status {
```

Comment:
The four selectors starting from this one are just moved here inside the `.wp-block-sensei-lms-course-outline-lesson`.